### PR TITLE
chore: pnpm override -> caniuse-lite: latest

### PR DESCRIPTION
**Pull request by insignias**
Issue #, if available:

Description of changes:

Checklist:

* [ ] Have you successfully deployed to an AWS account with your changes?
* [ ] Have you written new tests for your core changes, as applicable?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by aws-tyler — APPROVED** [review]

**Review by DaveMadsen — DISMISSED** [review]

File: .gitignore [hunk 1]
```diff
@@ -103,6 +103,9 @@ solutions/swb-reference/integration-tests/config/*.yaml
 solutions/swb-reference/src/config/*.json
 solutions/swb-reference/.aws-sam/
 
+#exclude binary files
+**/*.rpm
+
```
Comment (bhudnell) [inline .gitignore:108]: we probably shouldnt be ignoring all binaries, since there are some in SWB

**Review by bhudnell — CHANGES_REQUESTED** [review]
Remove the binary gitignore line

**Comment by bhudnell** [comment]
@SanketD92 is it ok that the two binary files were removed?

File: .gitignore (hunk 1 above)
Comment (insignias) [reply .gitignore:109]: @SanketD92 ok with this ?

**Review by aws-tyler — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (SanketD92) [reply .gitignore:109]: We only want to ignore binaries in this folder: `solutions/swb-reference/src/environment-files/offline-packages/sagemaker/fuse-2.9.4` if there are any here. 

The ones in boto3 and fuse-2.9.4_AL2 are still needed.

Comment (insignias) [reply .gitignore:109]: got it !

**Review by SanketD92 — APPROVED** [review]

**Review by DaveMadsen — APPROVED** [review]

**Review by bhudnell — APPROVED** [review]

**Pull request merged**